.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/c/sdks/5
+    about: Discuss this SDK in the Auth0 Community forums
+  - name: SDK API Documentation
+    url: https://auth0.github.io/auth0.net/
+    about: Read the API documentation for this SDK
```

Comment:
This repository doesn't have a FAQ or Library documentation, so I didn't include those.